corgidrp/recipe_templates/l1_to_boresight.json

```diff
@@ -0,0 +1,92 @@
+{
+    "name" : "l1_to_boresight",
+    "template" : true,
+    "drpconfig" : {
+        "track_individual_errors" : false
+    },
+    "inputs" : [],
+    "outputdir" : "",
+    "steps" : [
+        {
+            "name" : "prescan_biassub",
+            "keywords" : {
```

Comment:
I just made a change to this step in PR #192, so we should adopt it here eventually as well.